docs/src/components/ApidocLink/ApidocLink.js

```diff
@@ -105,11 +105,13 @@ export const createFromType = (type, defProps) => {
 const ApidocLink = ({ logo, renderLink }) => (
   <aside className={getClassName('bpkdocs-apidoc-link')}>
     {logo && (
+      /* eslint-disable backpack/use-components */
```

Comment:
Does this work? Might be neater?

```suggestion
      // eslint-disable-next-line backpack/use-components
```

Reply:
Ah yeah, probably. The thing is this is total laziness of me but I don't want to have to update all 20ish files to this format 🙈